### PR TITLE
ADDING login/signup functionality

### PR DESCRIPTION
**Pull request by Harshitraaaj**
generating new pull req , Fixing #1   kindly add labels in this new PR.

### CODE REVIEW

**Comment by vishal02527** [comment]
@Harshitraaaj are signup and login functionalities working efficiently now?
What other things do I need to take care of after merging it?

**Comment by Harshitraaaj** [comment]
@vishal02527  you have to follow these steps Process to run the program

step0: check whether your system already have nodejs , if Not installed then server will not run

step1: move to backend folder where main file index.js present { use this command : cd backend }

step2: create .env file and inside store { PORT = 5000 MONGODB_URL= mongodb+srv://username:@storage.eyb3g.mongodb.net/?retryWrites=true&w=majority&appName=storage

if you doesnt have mongodb atlas account make it and add URL here.

step3: install dependencies [ npm i bcrypt express express-session body-parser dotenv ... ] list of dependencies present in package.json

step:4 run in terminal node index.js [ inside backend file ]

step:5 open browser and type localhost:5000 [ your website is started 

 Simpliy run Index.js file the everything Is alredy setup ,

**Comment by Harshitraaaj** [comment]
Please add labels and assign me here and Please try to merge quickly . it reflects on my gssoc ranking .

File: frontend/signup.html [hunk 1]
```diff
@@ -110,21 +110,53 @@ <h2>Sign Up</h2>
           <button onclick="saveUsername()">Sign Up</button>
           <div class="login">
             Already have an account?
-            <a class="login-link" href="./login.html">Login</a>
+            <a class="login-link" href="./login_mongo.html">Login</a>
```
Comment (Harshitraaaj) [inline frontend/signup.html:113]: add this line else it doesnt call login page
```suggestion
            <a class="login-link" href="./login_mongo.html">Login</a>
```

**Comment by vishal02527** [comment]
@Harshitraaaj there are some conflicting files issues I can't merge it.

**Comment by rexx239** [comment]
@vishal02527  Which files and line no. ?

**Comment by Harshitraaaj** [comment]
@vishal02527  can you explain in detail what is the main issue , This are the list of conflicting files devotional-resources.html
dhanna-jatt.html
discussion-forum.html
divine-wallpapers.html
gallery.html
index.html
karmati-bai.html
meera-bai.html
narsi.html
raidas.html
scriptures.html
stories-of-devotees.html
surdas.html
taj-biwi.html right . just ignore my section of code and keep your code . its just resolve the merge conflict . until and unless login_mongo, signup.html and backend folder files is not altered login/signup function work properly .just merge from your side.

**Comment by Harshitraaaj** [comment]
@vishal02527 Have you checked ??

**Comment by Harshitraaaj** [comment]
@vishal02527 bhaii 2 week over but you didnt accept my PR , GSSOC EXT is going to over soon . Review and merge and add labels in this issue

**Comment by Harshitraaaj** [comment]
@vishal02527 broo what's the update regarding merge ?

**Comment by vishal02527** [comment]
@Harshitraaaj conflicting issues, I can't merge it.

**Comment by Harshitraaaj** [comment]
@vishal02527 now check is it resolve ?

**Pull request closed without merging**